library/pem.c

```diff
@@ -130,6 +142,120 @@ static int pem_pbkdf1( unsigned char *key, size_t keylen,
 
     return( ret );
 }
+#else
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Is this needed? Since `PEM_RFC1421` is defined `MBEDTLS_HAS_ALG_MD5_VIA_MD_OR_PSA_BASED_ON_USE_PSA` has to be defined.
So either we have MD5_C and the case above happens, or we don't, but `MBEDTLS_USE_PSA_CRYPTO` has to be on. Am I missing something?

Reply:
Yes, it can be removed.